testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergProcedureCalls.java

```diff
@@ -59,6 +59,98 @@ public void testMigrateHiveTable()
         onTrino().executeQuery("DROP TABLE IF EXISTS " + icebergTableName);
     }
 
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    public void testMigrateTimestampHiveTableWithOrc()
+    {
+        testMigrateTimestampHiveTable(
+                "MILLISECONDS",
+                "2021-01-01 10:11:12.123",
+                "2021-01-01 10:11:12.123000",
+                "2021-01-01 10:11:12.123",
+                "ORC");
+
+        testMigrateTimestampHiveTable(
+                "MICROSECONDS",
+                "2021-01-01 10:11:12.123456",
+                "2021-01-01 10:11:12.123456",
+                "2021-01-01 10:11:12.123456",
+                "ORC");
+
+        testMigrateTimestampHiveTable(
+                "NANOSECONDS",
+                "2021-01-01 10:11:12.123456789",
+                "2021-01-01 10:11:12.123457",
+                "2021-01-01 10:11:12.123456",
+                "ORC");
+    }
+
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    public void testMigrateTimestampMillisHiveTableWithAvro()
```

Comment:
Let's remove this test. An integration test (`testMigrateTimestampMillisTypeWithAvro`) is sufficient.

Reply:
done